load/load_distributor.go

```diff
@@ -26,20 +34,23 @@ func (ld *LoadDistributor) getOrCreatePartition(topic string) *LoadPartition {
 	using the load partition factory and then use that factory using the config and then
 	producing the channels*/
 
+	//add the desired config to the partition
 	config := LoadPartitionFactoryConfig{}
 
+	//creating channels for load partition
 	topicChannel := make(chan string)
 	consumerChannel := make(chan consumer.Consumer)
 
+	//creating a new load partition using load partiotion factory
 	factory := NewLoadPartitionFactory(&config)
 	newPartition := factory.Produce(topicChannel, consumerChannel)
 
-	//Adding the partition into the map and then returning
+	//Adding the partition into the map and then returning the new partition
 	TopicToPartitionMap[topic] = newPartition
 	return newPartition
 }
 
-func (ld *LoadDistributor) Listen() {
+func (ld *LoadDistributor) Start() {
```

Comment:
The code for `Start()` has to be fixed I think, any ideas on how we should approach this?